index.html

```diff
@@ -0,0 +1,141 @@
+  <meta property="og:image" content="./desktop_screenshot.png">
+  <title>Golden View School</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/f8b4998ade.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <div class="pre-nav">
+    <a href=""> <i class="fab fa-twitter"></i></a>
+    <a href=""><i class="fab fa-instagram"></i></a>
+    <ul class="pre-menu">
+      <li><a href="#">English</a></li>
+      <li><a href="#">My Page</a></li>
+      <li><a href="#">Logout</a></li>
+    </ul>
+  </div>
+  <nav id="nav" class="invisible">
+    <div id="cross" class="cross-container" onclick="toggleMenu()">
+      <i class="fas fa-times"></i>
+    </div>
+    <a href="./index.html" class="nav-logo-frame">
+      <img src="./assets/Golden-view.jpg" alt="" class="nav-logo">
+    </a>
+    <ul class="menu">
+      <li><a href="./about.html">About</a></li>
+      <li><a href="#" onclick="toggleMenu()">Program</a></li>
+      <li><a href="#" onclick="toggleMenu()">Join</a></li>
+      <li><a href="#" onclick="toggleMenu()">Sponsor</a></li>
+      <li><a href="#" onclick="toggleMenu()">News</a></li>
+    </ul>
+    <button type="button" class="nav-btn">Join Now!</button>
+    <div class="deco-ball-1"></div>
+    <img src="./assets/11-chess-board-gray.png" alt="" class="curtain-deco">
+  </nav>
+  <div class="img-frame">
+    <div id="burger-container" class="burger-container">
+      <i class="fas fa-bars"></i>
+    </div>
+    <img src="./assets/nairobi.jpg" alt="View of Nairobi City" class="back-img">
+    <div class="img-curtain">
+      <img src="./assets/11-chess-board-gray-resized.png" alt="" class="curtain-deco">
+      <img src="./assets/hitmark-gray.png" alt="" class="curtain-hitmark">
+    </div>
+  </div>
+  <header>
+    <h1>Hey There</h1>
+    <h2>Golden View <br>SChool<br>Welcomes you</h2>
+    <div class="deco-ball-1"></div>
+    <div class="txt-container">
+      <p>
+        This is Golden School Website. We are located in the Capital City Nairobi,Kenya.
+        We offer the best courses in different fields and we are ranked one of the best school in Nairobi.We offer our course both online and physical depending on what you will chose from. We offer many courses from Web-Program design,Database,server Courses, Linux Courses.You will Learn some of the Software Development skills needed like HTML, CSS, Javascript, Rust, Ruby,ReactJS, Python and many more.
+      </p>
+    </div>
+    <div class="tag-container">
+      <h3>2022.10.17(Monday) ~ 22(SAT)</h3>
+      <p>
+        @ Highway Towers Building , Nakuru Kenya
+      </p>
+      <img src="./assets/hitmark-gray.png" alt="" class="tag-hitmark">
+      <i class="fas fa-globe-americas"></i>
+    </div>
+  </header>
+  <main>
+    <h2>Main Program</h2>
```

Comment:
- [ ] I kindly suggest that you make sure that all the program articles are of the same size in width and length like it is in the Behance Design

|     |   |
| ---      | ---       |
| Behance Design | ![image](https://user-images.githubusercontent.com/52934128/196698982-3784840c-c21f-4144-8ada-ac35b24b47bc.png)      |
| Your Design   | <img width="940" alt="image" src="https://user-images.githubusercontent.com/52934128/196698116-771ac80e-9236-480c-8f40-437aece7215b.png">  |

- [ ] [OPTIONAL] I kindly suggest that you add the white border on hover.